drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/graal/lir/OCLLIRStmt.java

```diff
@@ -55,6 +55,37 @@ public final void emitCode(CompilationResultBuilder crb) {
 
     }
 
+    @Opcode("BEGIN_SKIP_SEQUENCE")
```

Comment:
What are these "skip" instructions? Shouldn't be call NOOP instructions instead? 
In which cases we emit these instructions?